lib/rucio/common/pcache.py

```diff
@@ -9,7 +9,7 @@
 import subprocess
 import signal
 
-from urllib import urlopen, urlencode
+from urllib import urlopen, urlencode  # not py3
```

Comment:
The issue why the module_importer fails is because of this. (On the py35 test)

Reply:
This is somewhat weird to me since urllib is imported as a dependency of requests, and requests is 35 compatible. So one option would be to just change these calls to requests.